Syntaxes/Processing.tmLanguage

```diff
@@ -99,204 +845,259 @@
+					<dict>
+						<key>0</key>
+						<dict>
+							<key>name</key>
+							<string>punctuation.definition.string.begin.processing</string>
+						</dict>
+					</dict>
+					<key>end</key>
+					<string>'</string>
+					<key>endCaptures</key>
+					<dict>
+						<key>0</key>
+						<dict>
+							<key>name</key>
+							<string>punctuation.definition.string.end.processing</string>
+						</dict>
+					</dict>
 					<key>name</key>
-					<string>constant.character.escape.java-processing</string>
+					<string>string.quoted.single.processing</string>
+					<key>patterns</key>
+					<array>
+						<dict>
+							<key>match</key>
+							<string>\\.</string>
+							<key>name</key>
+							<string>constant.character.escape.processing</string>
+						</dict>
+					</array>
 				</dict>
 			</array>
 		</dict>
+		<key>throws</key>
 		<dict>
 			<key>begin</key>
-			<string>'</string>
+			<string>throws</string>
 			<key>beginCaptures</key>
 			<dict>
 				<key>0</key>
 				<dict>
 					<key>name</key>
-					<string>punctuation.definition.string.begin.java-processing</string>
+					<string>storage.modifier.processing</string>
 				</dict>
 			</dict>
 			<key>end</key>
-			<string>'</string>
-			<key>endCaptures</key>
-			<dict>
-				<key>0</key>
-				<dict>
-					<key>name</key>
-					<string>punctuation.definition.string.end.java-processing</string>
-				</dict>
-			</dict>
+			<string>(?={|;)</string>
 			<key>name</key>
-			<string>string.quoted.single.java-processing</string>
+			<string>meta.throwables.processing</string>
 			<key>patterns</key>
 			<array>
 				<dict>
-					<key>match</key>
-					<string>\\.</string>
-					<key>name</key>
-					<string>constant.character.escape.java-processing</string>
+					<key>include</key>
+					<string>#object-types</string>
 				</dict>
 			</array>
 		</dict>
+		<key>values</key>
 		<dict>
-			<key>captures</key>
-			<dict>
-				<key>1</key>
-				<dict>
-					<key>name</key>
-					<string>storage.type.class.java-processing</string>
-				</dict>
-				<key>2</key>
+			<key>patterns</key>
+			<array>
 				<dict>
-					<key>name</key>
-					<string>entity.name.type.class.java-processing</string>
+					<key>include</key>
+					<string>#strings</string>
 				</dict>
-				<key>3</key>
 				<dict>
-					<key>name</key>
-					<string>storage.modifier.extends.java-processing</string>
+					<key>include</key>
+					<string>#object-types</string>
 				</dict>
-				<key>4</key>
 				<dict>
-					<key>name</key>
-					<string>entity.other.inherited-class.java-processing</string>
+					<key>include</key>
+					<string>#constants-and-special-vars</string>
 				</dict>
-			</dict>
-			<key>match</key>
-			<string>\b(class)\s+([a-zA-Z_](?:\w|\.)*)(?:\s+(extends)\s+([a-zA-Z_](?:\w|\.)*))?</string>
-			<key>name</key>
-			<string>meta.class.java-processing</string>
+			</array>
 		</dict>
-	</array>
+	</dict>
 	<key>scopeName</key>
-	<string>source.pde</string>
+	<string>source.processing</string>
```

Comment:
@kylefleming Can you explain the reason for this change? I believe this will affect the build system and other areas where we refer to `source.pde`.

/cc @b-g